src/probnum/prob/distributions/normal.py

```diff
@@ -398,7 +402,7 @@ def sample(self, size=()):
                                                    cov=self.cov(), size=size,
                                                    random_state=self.random_state)
 
-    def reshape(self, shape):
+    def reshape(self, newshape):
         raise NotImplementedError
 
     # Arithmetic Operations ###############################
```

Comment:
block comments seem inconsistent between different code pieces? Shall we use `# bla #########` or just `# bla`

Reply:
I vote for just `# ` or a standard based on n`#`s left and right of the comment.

Reply:
Yeah what happened in ` # Arithmetic Operations ############################### ` (and the respective other places) is one `#` on the left and as many on the right as necessary to get to 79 lines (to serve as visual separation between blocks of methods but still not longer than the default column width). I don't think this decision is any kind of bottleneck though so I don't really care too much.